### ISSUE TITLE
There is Ooops after importing a vm and the whole Virtual Machines doesn't work for lower browser version

### ISSUE THREAD

**Issue by cxhlunar** [cockpit-project/cockpit#14959]
Cockpit version: cockpit-232-1.fc33.x86_64
OS:Fedora33
Page: cockpit-machines-232-1.fc33.noarch
Chrome:Version 79.0.3945.88 (Official Build) (64-bit)
Firefox:62.0.3 (64-bit)

There is Ooops after importing a vm and the whole Virtual Machines doesn't work for lower browser version

P.S.:
Not sure whether it's the same root cause with #14936, let me file it here in case not
The problem is gone after upgrade Chrome to: Version 87.0.4280.66 (Official Build) (64-bit)

Steps to reproduce

1. Prepare Fedora33 and cockpit env, and an existing image:
/var/lib/libvirt/images/cirros.qcow2
2. Open Virtual Machines page, click Import VM and import the image in step 1, click import

There is Ooops , and the the whole Virtual Machines doesn't work, reboot or refresh doesn't help, the error message:
Refused to apply style from 'http://10.66.144.41:9090/cockpit/$5c00d0ee343098d381fda39264c84983b56c785f81d4c6d9091eca8f68647a5d/shell/nav.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
<xchen> ostree.js:303 Refused to apply inline style because it violates the following Content Security Policy directive: "default-src 'self' http://10.66.144.41:9090". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a 
 fallback.
 p @ ostree.js:303
 t.inject @ ostree.js:303
 t.a @ ostree.js:303
 (anonymous) @ ostree.js:3
 (anonymous) @ ostree.js:3
 n @ ostree.js:1
 (anonymous) @ ostree.js:327
 n @ ostree.js:1
 (anonymous) @ ostree.js:303
 n @ ostree.js:1
 (anonymous) @ ostree.js:1
 (anonymous) @ ostree.js:1
 cockpit.js:621 grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error
 p @ cockpit.js:621
m @ cockpit.js:631
 v @ cockpit.js:526
 a.onmessage.o.dispatch_data @ cockpit.js:456
 e @ cockpit.js:312
 postMessage (async)
 (anonymous) @ index.js:34
 a.onmessage.o.dispatch_data @ cockpit.js:454
 cockpit.js:621 Usage:
   ostree remote list [OPTION…]
 List remote repository names
 Help Options:
   -h, --help          Show help options
 Application Options:
   --repo=PATH         Path to OSTree repository (defaults to current directory or /sysroot/ostree/repo)
   -u, --show-urls     Show remote URLs in list
   -v, --verbose       Print debug information during command processing
   --version           Print version information and exit
 error: Command requires a --repo argument
 p @ cockpit.js:621
 m @ cockpit.js:631
 v @ cockpit.js:526
 a.onmessage.o.dispatch_data @ cockpit.js:456
 e @ cockpit.js:312
 postMessage (async)
 (anonymous) @ index.js:34
 a.onmessage.o.dispatch_data @ cockpit.js:454
 machines.js:345 TypeError: t.toLowerCase(...).replaceAll is not a function
     at Ks (machines.js:361)
     at Ga (machines.js:345)
     at _s (machines.js:345)
    at cl (machines.js:345)
     at sl (machines.js:345)
     at Zs (machines.js:345)
     at machines.js:345
     at t.unstable_runWithPriority (machines.js:353)
     at Ui (machines.js:345)
     at qi (machines.js:345)

![Screenshot from 2020-11-25 12-27-35](https://user-images.githubusercontent.com/12710195/100183354-ad1d5280-2f19-11eb-9c88-0769f0920856.png)

**Comment by KKoukiou** [comment]
Hey @cxhlunar you report two different issues here,

Thanks for posting the console output, it had enough information for me to understand where the crash happens, so I posted a blind solution PR here https://github.com/cockpit-project/cockpit/pull/14962.
Unfortunately I can't reproduce locally. Is that a consistent behavior on your system that importing VMs will cause an Oops?


Regarding the other issue, I am not sure what do you see. When the oops is gone, or let's say in some other page (not machines) does cockpit work as expected?

**Comment by cxhlunar** [comment]
Yes, actually I update my chrome to new version, so I imported a vm successfully now
Then I use firefox 62.0.3 (64-bit) to open the Virtual Machines page, it still got Ooops, so it's not only about importing I think, it can not display existing vms 

For other pages, it works fine, just you can still see the Ooops on old browser, but it can display the page ,not like machines - has blank page
I'll send you my env private.

**Comment by KKoukiou** [comment]
Reopening - since the PR which closed it addressed only the crash.
